### PR TITLE
Stop event and star checkbox

### PR DESCRIPTION
**Pull request by johangirod**

### CODE REVIEW

**Comment by cyrilf** [comment]
Seems good!

File: core/js/directives/StopEventDirective.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+angular.module('ev-fdm')
+    .directive('evStopEvent', function () {
```
Comment (christophehurpeau) [inline core/js/directives/StopEventDirective.js]: stop is not really explicit. It also could mean stopImmediatePropagation or maybe preventDefault ?
ev-stop-propagation might be more accurate ? (and more reflected to the dom api)
What do you think ?

Comment (johangirod) [inline core/js/directives/StopEventDirective.js]: Yes, you're right. I'll change it right away.

File: core/js/directives/StopEventDirective.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        return {
+            restrict: 'A',
+            link: function (scope, element, attr) {
+                attr.evStopEvent
+                    .split(',')
+                    .map(function (eventName) {
+                        return eventName.trim();
+                    })
+                    .forEach(function (eventName) {
+                        element.bind(eventName, function (e) {
```
Comment (christophehurpeau) [inline core/js/directives/StopEventDirective.js]: maybe you can do the trim here, no need to read to array twice

Comment (johangirod) [inline core/js/directives/StopEventDirective.js]: Well, that's right.

**Pull request merged**